src/app/app.component.ts

```diff
@@ -70,12 +71,19 @@ function fixUpNav(nav: any, baseUrl: string) {
   templateUrl: './app.component.html'
 })
 export class AppComponent implements OnInit {
+  public isReady = false;
+
   constructor(
     private router: Router,
     private windowRef: SkyAppWindowRef,
     private config: SkyAppConfig,
+    private styleLoader: SkyAppStyleLoader,
     @Optional() private searchProvider?: SkyAppSearchResultsProvider
-  ) { }
+  ) {
+    this.styleLoader.loadStyles().then(() => {
```

Comment:
From our previous conversation, my suggestion was something like this:

```
public hasError = false;
this.styleLoader.loadStyles.then(error => {
  this.isReady = true;
  if (error) {
    console.log(error);
    this.hasError = true;
  }
});
```

And then in `app.component.html`, something like:
```
<div *ngIf="hasError">
  There was an error loading the requested fonts.  The application is still available.
</div>
```